docs/platform_management_plan/software_verification.rst

```diff
@@ -477,6 +499,7 @@ section above). Driven by that the following test setups can be derived:
 #. GoogleTest
 #. Rust
 #. ITF
+#. Test Scenario Framework
```

Comment:
Do I see it correct that this is the first time mentioning ´Test Scenario Framework´? I may simply have missed it, but the current documentation of core cannot find the name: https://eclipse-score.github.io/score/main/search.html?q=Scenario#